solver/src/ethereum.rs

```diff
@@ -9,7 +9,10 @@ use primitive_types::H160;
 #[async_trait]
 pub trait SettlementContract {
     async fn get_nonce(&self, token_pair: TokenPair) -> Result<u32>;
-    async fn settle(&self, settlement: Settlement) -> Result<()>;
+    // Call simulates the transaction.
+    async fn settle_call(&self, settlement: &Settlement) -> Result<()>;
+    // Send really executes the transaction.
+    async fn settle_send(&self, settlement: &Settlement) -> Result<()>;
```

Comment:
ubernit: simulate_settle, execute_settle are maybe slightly more explicit what these methods do (for the caller it might be less important how they do it).

Reply:
I won't change the name now because it's just a placeholder trait until we have the settlement ethcontract contract.